website/src/pages/ComparePage/components/Hero.tsx

```diff
@@ -24,16 +24,16 @@ export default function Hero(props: { gitRef: any; setGitRef: any }) {
       <h1 className="mb-3 text-front text-opacity-70">
         Enter SHAs to compare commits
       </h1>
-      <div className="flex overflow-hidden bg-gradient-to-br from-primary to-theme p-[2px] rounded-full">
+      <div className="flex flex-col md:flex-row gap-4 overflow-hidden">
         <ComparisonInput
           name="old"
-          className="rounded-l-full"
+          className="text-xl px-6 py-4 bg-background border-2 border-gray-300 rounded-full focus:border-primary  focus:outline-none"
           setGitRef={setGitRef}
           gitRef={gitRef}
         />
         <ComparisonInput
           name="new"
-          className="rounded-r-full "
+          className="text-xl px-6 py-4 bg-background border-2 border-gray-300 rounded-full focus:border-primary focus:outline-none"
```

Comment:
In my opinion we should keep the same style for the input as what we have on other pages, here is a diff between search and compare:

<img width="812" alt="image" src="https://github.com/vitessio/arewefastyet/assets/35779988/bccfde31-bdb8-4440-a5b0-078c47e4daaf">
<img width="1126" alt="image" src="https://github.com/vitessio/arewefastyet/assets/35779988/b25ac22e-4aeb-4c5b-b30d-61c3484de5ed">

Reply:
got it, I'll make the input style consistent across pages 
what i suggest: instead of keeping separate input boxes, i'll follow the old style only and make that responsive by decreasing the size of that in small devices

Reply:
hey @frouioui , kindly check if this is fine by you and addresses the review comments correctly?

this fix brings responsiveness to compare page while keeping the input style consistent with older design.

Before : 
![11](https://github.com/vitessio/arewefastyet/assets/139503599/e401e598-4d40-46c9-b895-b93eea13766f)
After : 
![12](https://github.com/vitessio/arewefastyet/assets/139503599/cb8250ba-48a9-4ebe-8094-3e6231798005)